spec/services/group_notifier_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+describe GroupNotifier do
+  let(:current_user) { double(User, name: 'user', id: 1) }
```

Comment:
You can actually use existing FactoryBot values for user and group, but doubles do work too

Reply:
that's true, I usually prefer using doubles/mocks but if it's preferred I can use FactoryBot here

Reply:
Let's replace things with FactoryBot where it already exists. Thanks!

Reply:
Can do! I'll make those changes tonight